stubs/EloquentBuilder.stub

```diff
@@ -513,6 +513,15 @@ class Builder
      * @return static
      */
     public function whereRelation($relation, $column, $operator = null, $value = null);
+
+    /**
+     * Gets the result of a query in chunks.
+     *
+     * @param  int  $count
+     * @param  callable(\Illuminate\Database\Eloquent\Collection<TModelClass>, int): bool $callback
```

Comment:
`\Illuminate\Database\Eloquent\Collection` needs 2 generic types. I think the key here can be `array-key`

Reply:
Done!